src/convertToObject.js

```diff
@@ -5,8 +5,24 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const arrNamePlusValue = sourceString
+    .split(';')
+    .map((str) => str.trim())
+    .filter(Boolean);
+  const arrNameAndValue = arrNamePlusValue.map((str) => {
```

Comment:
Similarly, `arrNameAndValue` could be renamed to `splitKeyValuePairs` for better clarity.